cli/userInfo.go

```diff
@@ -0,0 +1,97 @@
+package main
```

Comment:
Sorry for nitpicking so many of your filenames. The reason I'm doing so is that having somewhat standard filenames will make it easier for a new developer to know where in the source code to look for specific functional chunks.
With that said could you rename this file to `handlers.go`, or something similar? `handlers` as a keyword is more typical to web APIs than CLIs but I'm blanking on a better word. The main idea is that this file contains the source code for actually `handling` a specific request made to the app, ex. `handle writing new metadata for a user`.